### ISSUE TITLE
Minimum votes required to implement accessibility  issue should be far less than normal issues.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#24642]
There are  a lot fewer number of assistive technology users  compared to normal users. So we should try to give level playing feild to assistive technology users by reducing the number of votes required to move it to implementation.There are less than 5% people only use assistive technologies.
